app/handlers/models.py

```diff
@@ -23,10 +23,15 @@
 
 @api.route("")
 class ModelList(Resource):
-
-    def get(self) -> Dict[str, Any]:
+    #@api.param("model_id", description="The model ID")
+    @api.marshal_with(model_metadata, code=200)
+    @api.response(400, "Invalid input")
+    def get(self, model_id: int) -> Dict[str, Any]:
         # TODO (jihyo): Function Comment
-        return {}
+        """Gets the list of modelMetadata"""
+        if model_id <= 0:
+            api.abort(400, "Invalid model ID")
+        return ModelService.get_model(model_id)
```

Comment:
Likewise, it should call a different function from `ModelService` (ex. `get_model_list`) that returns all the model metadata.